### PR TITLE
Mutant parts are removed if you roll a head job, as opposed to being barred from these jobs

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
add: People with mutant parts (cat ears) are no longer outright barred from selecting command roles in their preferences, but will have their mutant parts removed on spawning if they are selected for that role.
/:cl:

Why: Heads of staff slots often go unfilled and forcing people to choose between cosmetics and command roles probably plays a part in that.

### CODE REVIEW

**Comment by chanoc1** [comment]
stop

**Comment by Kraseo** [comment]
thanks kor

**Comment by chanoc1** [comment]
thank god

**Comment by Thunder12345** [comment]
@chanoc1 @optimumtact  This doesn't give cats ANY more rights, it just means they can roll heads and get their cat shit taken away if they get the job

**Comment by Cyberboss** [comment]
KORRRRRRRRRRRRRRRRRRR!!!!!!!!

**Comment by chanoc1** [comment]
@Thunder12345 thats what im saying catpeople hybrids are crimes against nature. im more for ligger rights than catpeople rights

**Comment by PKPenguin321** [comment]
This was actually a good not meme pr

**Comment by KorPhaeron** [comment]
@chanoc1 

Okay but this removes their cat ears

**Comment by chanoc1** [comment]
@KorPhaeron hmm but the cat character still remains...

**Comment by GunHog** [comment]
This is a perfectly reasonable PR that would serve to not bar head roles, which helps rounds. Also, for you human supremacists, this _reduces_ the number of cat-people, meaning that you should **support** this PR.

**Comment by optimumtact** [comment]
It encourages people to wear cat mutations because they can still roll for head roles.

This an unacceptable to my dog faction

**Comment by chanoc1** [comment]
^

**Comment by KorPhaeron** [comment]
@optimumtact 

You say that like being unable to choose a head role is some sort of punishment when they are already the least popular jobs

**Comment by lzimann** [comment]
if you want headroles just disable the ears simple

**Comment by KorPhaeron** [comment]
@lzimann 

If you want the ears gone just turn them off via config.

I'd do the same thing for lizards and plasmamen if their names would work properly when being turned human

**Comment by RemieRichards** [comment]
@lzimann this is exactly what this PR does, just automated for convenience.

**Comment by Kraseo** [comment]
the original name was better

**Comment by RemieRichards** [comment]
It was also misleading.

**Comment by GunHog** [comment]
Based RR saving the PR!

**Comment by Grazyn** [comment]
>I'd do the same thing for lizards and plasmamen if their names would work properly when being turned human

Give them a human name from the random list?

**Comment by Dorsisdwarf** [comment]
>Lzimann If you want the ears gone just turn them off via config.`

what an unprecedented idea

**Comment by Arianya** [comment]
This is an actual honest to god QoL fix.

**Comment by FantasticFwoosh** [comment]
Next up, a lizard who tries to sign up for a command role will be warned they will be turned into a random human with Proceed?/Return to menu prompt to stop lizard captains and maintain DEEPEST LORE.

**Comment by Cheridan** [comment]
>I'd do the same thing for lizards and plasmamen if their names would work properly when being turned human

Except people name their catgirls things like Kitty McMewmers so it'd be just as weird, nyaa.

**Comment by FantasticFwoosh** [comment]
@Cheridan this is partly the problem derived with RP'ing catpeople, they don't RP mutants (humans with screwed up ears) like they are supposed to, and instead roleplay full furries with silly names, personas and backstories like that and discriminate themselves from the 'Gajjin' of non-catpeople leading to cliques and confrontation we see within lizard race circles. Namely they calling out xenophobia and being targeted without reason or cause over a unalterable cosmetic choice that confuses everybody.

> TL;DR catpeople are race faking and making cliques, don't give them command jobs where they have executive power over a department. This is literally fur-face racism (adopting fur in order to mock a cat with racial traits) shame on you.

> I argue that this is a good change as the command staff need to be uniform and not give into this poisonous philosophy or get distracted being a 'cute kawaii cat' from important station duties, if i had thing my way id define them as a hard-race and leave it at the pro-human AI rules and company hierarchy to sort it out.

> **Everyday we stray further from gods light.**

> Catpeople are basically space age weabo

**Comment by Sligneris** [comment]
I have never seen a catperson with a name that couldn't be a human name.

Citation needed

**Comment by RemieRichards** [comment]
@FantasticFwoosh *gaikokujin please, gaijin has fallen out of favour for racist connotations.
Think "She was foreign" vs. "Fucking foreigners!"

**Comment by GunHog** [comment]
Given the context provided by FF and the nature of this community, the racist connotation is entirely appropriate! It is on the same station that adapted a human racist word to apply to an entire species!

**Comment by kevinz000** [comment]
Fwoosh shitposts are funny.

**Comment by Iamgoofball** [comment]
remove furry trash

**Comment by optimumtact** [comment]
@tgstation/commit-access someone else merge this or I'm closing it in 24 hours

**Comment by Jordie0608** [comment]
I don't see issue with this.
